lab-doug/package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Looks good!  Since you're using mocha for your tests you may want to change the "test" script to use the command "mocha" instead of "echo \"Error: no test specified\"  && exit 1", that way if someone were to call "npm test" on the command line it will automatically run mocha.